README.md

```diff
@@ -104,6 +105,19 @@ void Init()
     SceneCaptureComponent2D->CaptureSource = ESceneCaptureSource::SCS_FinalToneCurveHDR;
 }
 ```
+How to Set up NetImgui
+----------------------
+
+To use NetImgui, use content of the *net_imgui* branch in place of *master*. This will be gradually merged into the *master* but until then you need to use that experimental branch.
+
+Similarly like ImGui, NetImgui is built as part of the UnrealImGui plugin and no other integration steps are required.
+
+To be able to connect to the Unreal editor or application that use NetImgui, you need to run a server (netImguiServer). Please, see the [NetImgui](https://github.com/sammyfreg/netImgui) page for instructions how to get it.
+
+After launching the server for the first time, you need to add two client configurations, for ports 8889 and 8890. After that, you can either initialise connection from the server or set it to autoconnection mode. More info will be added later.
```

Comment:
Is this still true if `netImgui.cfg` is provided?

Reply:
You are right this is no longuer needed with provided config (the 2 basics config). Still need to add console config. It'll update this info.